src/test/java/com/epam/esm/config/TestConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.esm.config;
```

Comment:
not sure that test coverage s according to the task

Reply:
Rest assured it has a 91% coverage according to IntelliJ IDEA tools